### PR TITLE
Feature/shopping cart phase 1

### PR DESCRIPTION
**Pull request by amanpdyadav**
* Add Shopping cart to stepper
* Add translations for shopping cart strings
* Add shoppingCart component
* Update vehicle selector to use data from redux
* Add reducers to handle cart properties actions
* Add actions for setting cart properties
* Add properties to PermitCartState

### CODE REVIEW

File: src/i18n/en.json [hunk 1]
```diff
@@ -22,6 +22,19 @@
       "enterVehicleRegistrationNumber": "Enter the vehicle registration number",
       "returnToSelectAnAddress": "Return to select an address",
       "continue": "Continue"
+    },
+    "shoppingCart": {
+      "datePlaceHolder": "DD.MM.YYYY",
+      "durationOfParkingCode": "Duration of parking code",
```
Comment (mhieta) [inline src/i18n/en.json:28]: Is the **parking code** actually a **parking permit**? If so, we should probably use `parking permit` everywhere in the code.

File: src/i18n/fi.json [hunk 2]
```diff
@@ -22,6 +22,19 @@
...
+      "datePlaceHolder": "PP.KK.VVVV",
+      "durationOfParkingCode": "Pysäköintitunnuksen kesto",
+      "from": "Alkaen",
+      "immediately": "Alkaa heti",
+      "monthSelectionHelpText": "Enintään 12 kk",
+      "parkingSignStartDate": "Pysäköintitunnuksen alkamispäivä",
+      "price": "Hinta",
+      "subTitle": "Ajonuovosi",
+      "temporary": "Määräaikainen",
+      "timeBeing": "Toistaiseksi voimassaoleva",
```
Comment (mhieta) [inline src/i18n/fi.json:36]: `timeBeing` --> `openEnded`

File: src/i18n/fi.json [hunk 3]
```diff
@@ -22,6 +22,19 @@
...
+    "shoppingCart": {
+      "datePlaceHolder": "PP.KK.VVVV",
+      "durationOfParkingCode": "Pysäköintitunnuksen kesto",
+      "from": "Alkaen",
+      "immediately": "Alkaa heti",
+      "monthSelectionHelpText": "Enintään 12 kk",
+      "parkingSignStartDate": "Pysäköintitunnuksen alkamispäivä",
+      "price": "Hinta",
+      "subTitle": "Ajonuovosi",
+      "temporary": "Määräaikainen",
```
Comment (mhieta) [inline src/i18n/fi.json:35]: `temporary`--> `fixedPeriod`

File: src/i18n/fi.json [hunk 4]
```diff
@@ -22,6 +22,19 @@
...
       "returnToSelectAnAddress": "Palaa valitsemaan osoite",
       "continue": "Jatka"
+    },
+    "shoppingCart": {
+      "datePlaceHolder": "PP.KK.VVVV",
+      "durationOfParkingCode": "Pysäköintitunnuksen kesto",
+      "from": "Alkaen",
+      "immediately": "Alkaa heti",
+      "monthSelectionHelpText": "Enintään 12 kk",
+      "parkingSignStartDate": "Pysäköintitunnuksen alkamispäivä",
```
Comment (mhieta) [inline src/i18n/fi.json:32]: `parkingSignStartDate` --> `parkingPermitStartDate`

File: src/i18n/fi.json [hunk 5]
```diff
@@ -22,6 +22,19 @@
...
+      "price": "Hinta",
+      "subTitle": "Ajonuovosi",
```
Comment (mhieta) [inline src/i18n/fi.json:34]: Typo: `Ajonuovosi` --> `Ajoneuvosi`

File: src/i18n/fi.json [hunk 6]
```diff
@@ -22,6 +22,19 @@
       "enterVehicleRegistrationNumber": "Syötå ajoneuvon rekisteritunnus",
```
Comment (mhieta) [inline src/i18n/fi.json:22]: Typo: `Syötå ajoneuvon rekisteritunnus` --> `Syötä ajoneuvon rekisteritunnus`

**Review by mhieta — CHANGES_REQUESTED** [review]
In general looks good, but added some comments regarding namings that should be still fixed before merging.

File: src/i18n/fi.json (hunk 5 above)
Comment (amanpdyadav) [reply src/i18n/fi.json]: Done

File: src/i18n/fi.json (hunk 4 above)
Comment (amanpdyadav) [reply src/i18n/fi.json]: Done

File: src/i18n/fi.json (hunk 3 above)
Comment (amanpdyadav) [reply src/i18n/fi.json]: Done

File: src/i18n/fi.json (hunk 2 above)
Comment (amanpdyadav) [reply src/i18n/fi.json]: Done

File: src/i18n/en.json (hunk 1 above)
Comment (amanpdyadav) [reply src/i18n/en.json]: Done

File: src/i18n/fi.json (hunk 6 above)
Comment (amanpdyadav) [reply src/i18n/fi.json]: Done

File: src/i18n/en.json (hunk 1 above)
Comment (amanpdyadav) [reply src/i18n/en.json]: Done

File: src/i18n/fi.json (hunk 2 above)
Comment (amanpdyadav) [reply src/i18n/fi.json]: Done

File: src/i18n/fi.json (hunk 3 above)
Comment (amanpdyadav) [reply src/i18n/fi.json]: Done

File: src/i18n/fi.json (hunk 4 above)
Comment (amanpdyadav) [reply src/i18n/fi.json]: Done

File: src/i18n/fi.json (hunk 5 above)
Comment (amanpdyadav) [reply src/i18n/fi.json]: Done

**Review by mhieta — APPROVED** [review]
All Good now 💯

**Pull request merged**